libs/net/dal/Models/ContentFilter.cs

```diff
@@ -145,6 +145,9 @@ public class ContentFilter : PageFilter
     /// </summary>
     public string? Keyword { get; set; }
 
+    // get/set - Get/set the namess to search for (tied to ministers).
```

Comment:
Spelling `namess`.  What's the difference between `Names` and `Aliases`?

Reply:
good catch on the `namess` typo.

Names requires a different type of search than just keyword and the name is now used to dynamically generate the alias on the fly.  Had a conversation with the business yesterday and the dynamic alias we generate is sufficient for now.  Having said that, in some future state we may re-use the alias field for configuring more synonyms in elasticsearch.